### DISCUSSION TITLE
Meet the Deepgram DevRel Team!

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#2] [General help]
**Luke Oliff**, Senior Developer Experience Engineer. [@lukeocodes](https://github.com/lukeocodes)  is an experienced technology content creator. He is a dad of two, tech streamer on Twitch, and open source contributor/maintainer. You can catch him leading our DevEx team’s unified web project or learn more about his history in his [welcome post](https://developers.deepgram.com/blog/2021/11/luke-oliff-joins-deepgram/).

**Sandra Rodgers**, Developer Experience Engineer. [@SandraRodgers](https://github.com/SandraRodgers) spent 10 years as a high school and middle school teacher before transitioning to a career in software engineering. On the developer relations team at Deepgram, she blends both of her professional passions - education and development. Find out more about her tiny dogs or the inspiration behind her career change [here](https://developers.deepgram.com/blog/2021/11/meet-sandra-rodgers/).

**Brian Barrow**, Developer Experience Engineer. [@BrianBarrow](https://github.com/briancbarrow ) is an avid disc golfer, a proud father of two, and a vegan. You can find out more about his involvement in FreeCodeCamp and what he does in his free time in his [welcome post](https://developers.deepgram.com/blog/2021/11/brian-barrow-hello/).

**Comment by lukeocodes** [comment]
Hello! :D

**Comment by briancbarrow** [comment]
Hey everyone!

**Comment by phazonoverload** [comment]
Sup sup!

**Comment by SandraRodgers** [comment]
Hi everyone! Happy to welcome you to the community! I hope we can provide support and improve your developer experience as much as possible. Please reach out, even just to say hi!

**Comment by michaeljolley** [comment]
Hello world! Welcome to the DG community discussions. We're all here to help, chat, make dad-jokes, etc. 😁

**Comment by geekchick** [comment]
Hey everyone! Looking forward to getting to know you in our Deepgram community!
